sdk/go/dagger.gen.go

```diff
@@ -8115,57 +8115,99 @@ func (CacheSharingMode) IsEnum() {}
 
 const (
 	// Shares the cache volume amongst many build pipelines, but will serialize the writes
-	Locked CacheSharingMode = "LOCKED"
+	CacheSharingModeLocked CacheSharingMode = "LOCKED"
+
+	// Shares the cache volume amongst many build pipelines, but will serialize the writes
+	// Deprecated: use CacheSharingModeLocked instead
+	Locked CacheSharingMode = CacheSharingModeLocked
+
+	// Keeps a cache volume for a single build pipeline
+	CacheSharingModePrivate CacheSharingMode = "PRIVATE"
 
 	// Keeps a cache volume for a single build pipeline
-	Private CacheSharingMode = "PRIVATE"
+	// Deprecated: use CacheSharingModePrivate instead
+	Private CacheSharingMode = CacheSharingModePrivate
+
+	// Shares the cache volume amongst many build pipelines
+	CacheSharingModeShared CacheSharingMode = "SHARED"
 
 	// Shares the cache volume amongst many build pipelines
-	Shared CacheSharingMode = "SHARED"
+	// Deprecated: use CacheSharingModeShared instead
+	Shared CacheSharingMode = CacheSharingModeShared
 )
 
 type ImageLayerCompression string
 
 func (ImageLayerCompression) IsEnum() {}
 
 const (
-	Estargz ImageLayerCompression = "EStarGZ"
+	ImageLayerCompressionEstargz ImageLayerCompression = "EStarGZ"
 
-	Gzip ImageLayerCompression = "Gzip"
+	// Deprecated: use ImageLayerCompressionEstargz instead
+	Estargz ImageLayerCompression = ImageLayerCompressionEstargz
 
-	Uncompressed ImageLayerCompression = "Uncompressed"
+	ImageLayerCompressionGzip ImageLayerCompression = "Gzip"
 
-	Zstd ImageLayerCompression = "Zstd"
+	// Deprecated: use ImageLayerCompressionGzip instead
+	Gzip ImageLayerCompression = ImageLayerCompressionGzip
+
+	ImageLayerCompressionUncompressed ImageLayerCompression = "Uncompressed"
+
+	// Deprecated: use ImageLayerCompressionUncompressed instead
+	Uncompressed ImageLayerCompression = ImageLayerCompressionUncompressed
+
+	ImageLayerCompressionZstd ImageLayerCompression = "Zstd"
+
+	// Deprecated: use ImageLayerCompressionZstd instead
+	Zstd ImageLayerCompression = ImageLayerCompressionZstd
 )
 
 type ImageMediaTypes string
 
 func (ImageMediaTypes) IsEnum() {}
 
 const (
-	Dockermediatypes ImageMediaTypes = "DockerMediaTypes"
+	ImageMediaTypesDockermediatypes ImageMediaTypes = "DockerMediaTypes"
+
+	// Deprecated: use ImageMediaTypesDockermediatypes instead
+	Dockermediatypes ImageMediaTypes = ImageMediaTypesDockermediatypes
```

Comment:
Why isn't this `DockerMediaTypes` and `ImageMediaTypesDockerMediaTypes`? Can you take the opportunity to fix the casing in the new scoped one? I've seen other instances of this. `ImageMediaTypesOcimediatypes` could be `ImageMediaTypesOCIMediaTypes`.

Reply:
Pushed a commit to resolve this - I've completely changed this implementation for the new enum names, now we actually try and avoid explicitly removing all the casing.

All the remaining case issues are an artifact of #8122.